src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php

```diff
@@ -372,7 +372,7 @@ protected function getUser()
     protected function isCsrfTokenValid(string $id, string $token): bool
     {
         if (!$this->container->has('security.csrf.token_manager')) {
-            throw new \LogicException('CSRF protection is not enabled in your application.');
+            throw new \LogicException('CSRF protection is not enabled in your application. Enable it with the "csrf_protection" key in "config/packages/framework.yaml"');
```

Comment:
`Enable them in config/packages/framework.yaml.` vs. `Enable it with the "csrf_protection" key in "config/packages/framework.yaml"`

always be specific/generic?